app/code/Magento/Theme/view/frontend/layout/default_head_blocks.xml

```diff
@@ -18,7 +18,6 @@
                     <argument name="criticalCssViewModel" xsi:type="object">Magento\Theme\Block\Html\Header\CriticalCss</argument>
                 </arguments>
             </block>
-            <block name="css_rel_preload_script" ifconfig="dev/css/use_css_critical_path" template="Magento_Theme::js/css_rel_preload.phtml"/>
```

Comment:
Unfortunately this is backward incompatible change that leads to SVC failure which is not allowed in patch releases. Could you revert block removal and just put the empty view file instead with only deprecation notice, so it will not output anything?

Reply:
i can remove this block in layout xml so we don't need to empty template
Could add some note in layout for later refactor

Reply:
good idea, but block with name `css_rel_preload_script` should be there.

Reply:
yes i have updated in latest commit

Reply:
I think even adding remove statement could break customizations, I suggest to go with one of those solutions:
- update view to not output anything
- remove “template” from the block

Reply:
ok i have removed it from template now deprecated template will not output anything